development/src/composables/sessionManagementFunction.js

```diff
@@ -0,0 +1,124 @@
+import { generatePeerIdCharacter } from "./peerIdGenerator";
+import { toast } from "react-toastify";
```

Comment:
we are no longer using the `react-toastify`
you need to utilize the `errorModal_comp`

Reply:
Aight